hazelcast-sql/src/test/java/com/hazelcast/jet/sql/SqlTestSupport.java

```diff
@@ -248,6 +253,16 @@ public static String hadoopNonExistingPath() {
                 : "/non/existing/path";
     }
 
+    public static Job awaitSingleRunningJob(HazelcastInstance hz) {
```

Comment:
Maybe a better place for it is `JetTestSupport`.

Reply:
Totally agree!